### ISSUE TITLE
[BUG] [Linter] H008 false positive on single quotes inside a template-tag string argument

### ISSUE THREAD

**Issue by PetrDlouhy** [djlint/djLint#2247]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Ubuntu 24.04
 - Python Version (``python --version``): Python 3.12.3
 - djLint Version (``djlint --version``): version 1.40.8
 - template language: Django

## Issue

H008 ("Attributes should be double quoted") fires on single quotes that are inside a **string argument to a template tag**, not on an actual HTML attribute:

```htmldjango
{% bootstrap_button button_type="submit" content="<i class='fas fa-edit'></i> Save" %}
```

`djlint --lint --profile=django` reports:

```
H008 1:50 Attributes should be double quoted. <i class='fas fa-edi
```

Here `content="..."` is a keyword argument to the `{% bootstrap_button %}` tag (correctly double-quoted). The `<i class='...'>` is HTML *inside that string value*; the single quotes are deliberate so they don't clash with the surrounding double quotes. They are not an HTML attribute in the document, so H008 should not apply — djLint is linting HTML-like content inside a quoted template-tag argument.

(#706 is a related but different request — auto-fixing genuinely single-quoted attributes. This is about not flagging single quotes that live inside a template-tag string argument at all.)

## How To Reproduce

Save the snippet as `repro.html` and run:

```
djlint --lint --profile=django repro.html
```
